src/app/pages/TokenDashboardPage/TokenSupplyCard.tsx

```diff
@@ -32,7 +32,16 @@ export const TokenSupplyCard: FC = () => {
                 color: COLORS.brandDark,
               }}
             >
-              {!supplyString ? t('common.missing') : t('tokens.totalSupplyValue', { value: supplyString })}
+              {!supplyString
+                ? t('common.missing')
+                : t('common.valuePair', {
+                    value: supplyString,
+                    formatParams: {
+                      value: {
+                        notation: 'compact',
+                      } satisfies Intl.NumberFormatOptions,
+                    },
+                  })}
```

Comment:
So this format is different than what we are trying to use in #703, although it's the very same data. I think we should use the same formatting everywhere...

Reply:
yeah I agree it should be the same. Will talk with Don tomorrow.